src/devtools.ts

```
@@ -21,6 +21,7 @@ window.inspectPlayer = () => require('fs').promises.readFile('/world/playerdata/
 
 Object.defineProperty(window, 'debugSceneChunks', {
   get () {
+    if (!(viewer.world instanceof WorldRendererThree)) return
```

ESLint

[error] Expected to return a value in method 'get'.

(getter-return)